plutus-book/doc/marlowe/marlowe_doc.adoc

```diff
@@ -17,108 +17,6 @@ enough to build every kind of contract found in the ACTUS classification
 of financial contracts (we shall see examples of building some of these later
 on).
 
-The Marlowe DSL is a tool for building smart contracts.
```

Comment:
Polina did add a bunch of literate content to the old marlowe interpreter, it would be nice not to lose it. Do we need to delete those files in this PR? They should be unrelated, since they're not involved in the playground, right?